File: FootballPredictor/Models/Fixtures/FixtureScore.cs
using FootballPredictor.Models.Competitions;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace FootballPredictor.Models.Fixtures
{
    public class FixtureScore : IFixtureScore
    {
        public byte HomeGoals { get; protected set; }
        public byte AwayGoals { get; protected set; }
        public string EnteredScore
        {
            get
            {
                return string.Format("{0}-{1}", HomeGoals, AwayGoals);
            }
        }


        public FixtureScore(byte homeGoals, byte awayGoals)
        {
            HomeGoals = homeGoals;
            AwayGoals = awayGoals;
        }

    }
}

File: FootballPredictor/Scripts/Models/Fixtures/FixtureScore.ts
namespace Models.Fixtures {
    export class FixtureScore {
        id: number;
        homeGoals: number;
        awayGoals: number;
        completed: boolean;

        constructor(id: number, homeGoals: number, awayGoals: number, completed: boolean) {
            this.id = id;
            this.homeGoals = homeGoals;
            this.awayGoals = awayGoals;
            this.completed = completed;
        }
    }
}